styles.css

```diff
@@ -0,0 +1,382 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  color: #333;
+  text-decoration: none;
+}
+
+footer div div a {
+  flex-basis: 24px;
+}
+
+.content-container a {
+  color: #316891;
+  text-decoration: underline;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+nav a:hover {
+  text-decoration: none;
+}
+
+nav {
+  padding: 5px 16px;
+  border-bottom: 1px solid lightgray;
+  position: fixed;
+  width: 100%;
+  background-color: white;
+}
+
+#left-nav {
+  flex-basis: 250px;
+}
+
+#logo {
+  width: 190px;
+}
+
+#right-nav {
+  flex-basis: 178px;
+}
+
+button {
+  background-color: #567b95;
+  border: none;
+  padding: 10px 13px;
+  color: #fff;
+  border-radius: 3px;
+  font-size: x-small;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #316891;
+}
+
+#left-nav button {
+  background-color: transparent;
+  border: none;
+  color: #333;
+  font-size: medium;
+}
+
+h1 {
+  padding: 88px 0 24px;
+  font-size: 2.3rem;
+  font-family: "Tinos", "Quattrocento", sans-serif;
+  font-style: italic;
+}
+
+h1,
+.content-container {
+  margin: 0 auto;
+  width: 600px;
+}
+
+h2 {
+  margin: 32px 0 8px;
+  font-size: 22px;
+  color: #333;
+}
+
+section figure figcaption {
+  width: 742px;
+  font-family: "Tinos", "Quattrocento", sans-serif;
+}
+
+.image-container {
+  width: 960px;
+  padding-bottom: 40px;
+}
+
+.ep-image {
+  width: 80px;
+  margin: 8px 8px;
+}
+
+.content-container {
+  width: 600px;
+}
+
+.full-width {
+  width: 100%;
+}
+
+iframe {
+  border: none;
+}
+
+figure {
+  margin: 32px auto;
+}
+
+.content-container p {
+  padding: 10px 0;
+  font-size: 20px;
+  color: #333;
+  font-family: "Source Serif Pro", "Libre Caslon Text", sans-serif;
+}
+
+#inflation-section {
+  width: 944px;
+  margin-bottom: 40px;
+}
+
+main hr {
+  width: 72px;
+  background-color: hsl(0, 0%, 90%);
+  border: none;
+  height: 1.3px;
+}
+
+aside {
+  float: right;
+  width: 250px;
+  margin: -100px 80px -350px 0;
+}
+
+aside hr {
+  width: 100%;
+  margin: 10px 0;
+}
+
+#inflation-grid {
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 16px;
+}
+
+footer hr {
+  margin: 10px 0;
+  border: none;
+  border-top: 1px solid hsl(0, 0%, 85%);
+}
+
+footer hr:first-child {
+  height: 3.5px;
+  border: 1px solid hsl(0, 0%, 70%);
+  border-left: none;
+  border-right: none;
+  margin: 16px 0;
+}
+
+#inflation-section hr {
+  border: 1px solid #333;
+}
+
+#suggestion-section hr {
+  border: none;
+  height: 1px;
+  background-color: #909090;
+}
+
+#comment-button {
+  padding: 12px 104px;
+  font-size: smaller;
+  font-weight: bold;
+}
+
+#social-bottom {
+  margin: 20px 0 64px;
+}
+
+#suggestion-section {
+  width: 1200px;
+  grid-template-columns: 3fr 1fr;
+  grid-gap: 24px;
+  margin-bottom: 100px;
+}
+
+footer div div a:hover {
+  text-decoration: underline;
+}
```

Comment:
This selector has the same style definition of selection on line 21. Group selectors with the same style definition.